test

```diff
@@ -0,0 +1,40 @@
+
+abi <abi/3.0>,
```

Comment:
Tox is incorrectly creating this file. We need to address this in a future PR, but let's just remove that file now

Reply:
Arg sorry I missed this

Reply:
Done!